<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemberOfCircleDialog</class>
 <widget class="QDialog" name="MemberOfCircleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>421</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>451</width>
    <height>421</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>451</width>
    <height>421</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>동아리 멤버 조회</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <spacer name="verticalSpacer_7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_9">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>1</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">font: 11pt &quot;나눔바른고딕&quot;;</string>
             </property>
             <property name="text">
              <string>학번 : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="stuNumLabel">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 11pt &quot;나눔바른고딕&quot;;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>1</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true">font: 11pt &quot;나눔바른고딕&quot;;</string>
             </property>
             <property name="text">
              <string>이름 : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="nameLabel">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 11pt &quot;나눔바른고딕&quot;;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_9">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <spacer name="horizontalSpacer_13">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>1</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="styleSheet">
              <string notr="true">font: 11pt &quot;나눔바른고딕&quot;;</string>
             </property>
             <property name="text">
              <string>학년 : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="stuYearLabel">
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 11pt &quot;나눔바른고딕&quot;;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>1</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="styleSheet">
              <string notr="true">font: 11pt &quot;나눔바른고딕&quot;;</string>
             </property>
             <property name="text">
              <string>학과 : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="departmentLabel">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 11pt &quot;나눔바른고딕&quot;;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>1</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="styleSheet">
              <string notr="true">font: 11pt &quot;나눔바른고딕&quot;;</string>
             </property>
             <property name="text">
              <string>전화번호 : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="phoneNumLabel">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 11pt &quot;나눔바른고딕&quot;;</string>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_11">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="verticalSpacer_10">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>3</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;나눔바른고딕&quot;;</string>
         </property>
         <property name="text">
          <string>지원동기</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>348</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="minimumSize">
        <size>
         <width>411</width>
         <height>51</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>411</width>
         <height>51</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;나눔바른고딕&quot;;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>3</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;나눔바른고딕&quot;;</string>
         </property>
         <property name="text">
          <string>앞으로의 다짐</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>318</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit_2">
       <property name="minimumSize">
        <size>
         <width>411</width>
         <height>71</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>411</width>
         <height>71</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;나눔바른고딕&quot;;</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>8</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="beAdminButton">
           <property name="minimumSize">
            <size>
             <width>91</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>91</width>
             <height>31</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 9pt &quot;나눔바른고딕&quot;;</string>
           </property>
           <property name="text">
            <string>관리자 승격</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteMemberButton">
           <property name="minimumSize">
            <size>
             <width>91</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>91</width>
             <height>31</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 9pt &quot;나눔바른고딕&quot;;</string>
           </property>
           <property name="text">
            <string>멤버 강퇴</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>188</width>
           <height>68</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="approvalButton">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>31</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 9pt &quot;나눔바른고딕&quot;;</string>
           </property>
           <property name="text">
            <string>승인</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="disapprovalButton">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>31</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 9pt &quot;나눔바른고딕&quot;;</string>
           </property>
           <property name="text">
            <string>탈락</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="cancelButton">
           <property name="minimumSize">
            <size>
             <width>111</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>111</width>
             <height>31</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 9pt &quot;나눔바른고딕&quot;;</string>
           </property>
           <property name="text">
            <string>나가기</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_10">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
